presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5341,4 +5341,12 @@ public void testMultipleSqlFunctionsWithLambda()
                 "SELECT array_sum(zip_with(a, b, (x, y) -> x * y)), array_sum(zip_with(a, b, (x, y) -> x * y)) + array_sum(zip_with(a, a, (x, y) -> x * y)) FROM (VALUES (ARRAY[1, 2, 3], ARRAY[1, 0, 0])) t(a, b)",
                 "SELECT 1, 15");
     }
+
+    @Test
+    public void testDistinctFrom()
```

Comment:
Can we add more tests to cover all code paths (both left and right)? Also since we are adding a generic test for `IS DISTINCT FROM` here, probably should also test for other common scenarios.

Reply:
@rongrong I have now added several more tests. Tests for basic cases are already in place in https://github.com/prestodb/presto/blob/0cfa23fd729a/presto-main/src/test/java/com/facebook/presto/sql/TestExpressionInterpreter.java#L226 . Therefore I would like to focus only on nested queries over here.